### DISCUSSION TITLE
rule: disallow `ingress` & `egress` on `aws_security_group`, prefer using `aws_security_group_rule`

### DISCUSSION THREAD

**Discussion by drewmullen** [terraform-linters/tflint-ruleset-aws#419] [Ideas]
Many issues can arise from embedding rules in `aws_security_group`. To avoid, should disallow the keys `ingress` and `egress` and point them to using `aws_security_group_rule`

**Comment by kayman-mk** [comment]
Looks like they changed to `aws_vpc_security_group_egress_rule` and `aws_vpc_security_group_ingress_rule`.

**Reply by kayman-mk** [reply]
Tackled in #793
